_commontech/general_product_overview.md

```diff
@@ -10,19 +10,19 @@ author: Leslie Lundquist
 
 ### General Product Overview
 
-Both offerings of **IBM Blue Box Cloud**, Dedicated and Local, are deployed automatically through a **Site Controller** machine, which serves as a storehouse for the **Ursula/Ansible playbooks** we use to create your cloud. You can view the [Ursula playbooks](https://github.com/blueboxgroup/ursula) on **GitHub**. They are open source documents.
+Both offerings of **IBM Bluemix on OpenStack**, Dedicated and Local, are deployed automatically through a **Site Controller** machine, which serves as a storehouse for the **Ursula/Ansible playbooks** we use to create your cloud. You can view the [Ursula playbooks](https://github.com/blueboxgroup/ursula) on **GitHub**. They are open source documents.
 
 **Site Controller**
 
-To deploy the **IBM Blue Box Dedicated** offering, a physical **Site Controller** machine must be located in a **SoftLayer Datacenter** within the same geographical region in which your new IBM Blue Box Dedicated cloud is deployed.
+To deploy the **IBM Bluemix Dedicated on OpenStack** offering, a physical **Site Controller** machine must be located in a **SoftLayer Datacenter** within the same geographical region in which your new IBM Bluemix Dedicated on OpenStack is deployed.
 
-To deploy the **IBM Blue Box Local** cloud offering, a Site Controller logical machine must be co-located within your IBM Local Cloud data center, and it will communicate with a Central Site Controller machine at IBM Blue Box.
+To deploy the **IBM Bluemix Local on OpenStack** cloud offering, a Site Controller logical machine must be co-located within your IBM Local Cloud data center, and it will communicate with a Central Site Controller machine at IBM Bluemix on OpenStack.
```

Comment:
Please refer to PR #421 for the correct wording of the following phrase. Thanks!

...it will communicate with a Central Site Controller machine at IBM Bluemix on OpenStack.